tests/unit/dataactbroker/test_fabs_derivations.py

```diff
@@ -294,7 +289,23 @@ def test_legal_country(database):
 
     obj = initialize_test_obj(legal_country='NK')
     obj = fabs_derivations(obj, database.session)
-    assert obj['legal_entity_country_name'] is None
+    assert not obj['legal_entity_country_name']
+
+
+def test_primary_place_county(database):
```

Comment:
We're already deriving/testing `place_of_perform_county_na` in a higher test (`test_ppop_derivations`) so this new test is unnecessary. You also removed all the test lines from that function for `place_of_perform_county_co` so they need to go back in.